### ISSUE TITLE
[Deque Analysis] Edit Item/Collection "critical" accessibility issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1172]
> Critical: This issue results in blocked content for people with disabilities, and will definitely prevent them from accessing fundamental features or content. This type of issue puts your organization at risk. Prioritize fixing as soon as possible, within the week if possible. Remediation should be a top priority. 

## Deque Analysis Summary

A total of 5 Critical issues exist related to editing Items or Collections. They are summarized here:

1. ~~(Issue ID: 470946) Edit Item, Bitstreams tab: "Content should be in a data table but is not."  The page appears visually as a table, but it is not represented as a `<table>`.~~ MOVED TO #1270 
2. (Issue ID: 471444) Edit Item, Metadata tab: "Label in `<th>` is not associated with form control."  The row and column header cells visually serve as labels for the form controls contained in the table. The form controls are not marked up in a manner that exposes this relationship.
    * Fix this issue by using an aria-labelledby attribute on the <input> to reference the label in another cell. The value of the aria-labelledby attribute is the id attribute value of the visible text label.
    * IMPORTANT: Note that the table headers/labels must be in <span> elements and that it is the <span> element and NOT the <th> element that has the id. If you place the id on the <th> element, some screen readers will not read the labels correctly when you tab through the form elements.
3. ~~(Issue ID: 470936) Edit Item, Bitstreams tab: Button to reorder bitstreams (in a bundle) is not keyboard accessible.~~ MOVED TO #1271 
4. (Issue ID: 470091) Edit Collection/Community, Logo upload is not keyboard accessible
    * NOTE: This same issue appears in uploading files in Submission, see #1171 & it should be solved similarly.
5. (Issue ID: 471077) Edit Collection/Community, Logo/file delete button (red trash icon) does not have a name/label.
    * NOTE: This same issue appears in Submission after uploading files. See #1171 & it should be solved similarly.

The full list of these issues (which includes links for more info) can be found at this link (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=100&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=4&filter%5Bpage_number%5D=12&filter%5Bpage_number%5D=13&filter%5Bpage_number%5D=14&filter%5Bpage_number%5D=15

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by tdonohue** [comment]
@atarix83 ; Assigning this one to you/your team for analysis as well, as several issues overlap with the Submission accessibility ticket (#1171)... So, it may be possible to solve both together, or take what you learn from the Submission fixes & reapply them here.

**Comment by atarix83** [comment]
Here the critical point is that the item's bitstream tab list has a draggable list. I estimate about 3/4 hours to fix all them

**Comment by tdonohue** [comment]
Thanks @atarix83 ! I'll add that estimate & assign to you for your team to work on (when you are ready)
